src/app.js

```diff
@@ -1,13 +1,28 @@
 import express, { json, urlencoded } from 'express';
 import cors from 'cors'
 import morgan from 'morgan'
+import bodyParser from 'body-parser'
+import mongoose from 'mongoose'
+//import route from './routes/index';
 
 const app = express();
+mongoose.connect('mongodb+srv://grace:bIwdzDRydpY28vZc@cluster0-n7fk2.mongodb.net/test?retryWrites=true&w=majority')
```

Comment:
Extract this to a config file and put the mongodb url in the environment variables